### PR TITLE
Resolve dependabot alerts

### PR DESCRIPTION
**Pull request by vsvipul**
This should resolve 3 dependabot alerts- 
- ansi-regex - Fixed by updating jest 
- json-schema - Automatically removed after updating jest
- node-notifier -  Automatically removed after updating jest

node-fetch is one dependency that is dependent on toolkit. Will need to fix this there and then update actions/cache version here to fix that.

### CODE REVIEW

File: jest.config.js [hunk 1]
```diff
@@ -17,7 +17,7 @@ const processStdoutWrite = process.stdout.write.bind(process.stdout);
 process.stdout.write = (str, encoding, cb) => {
     // Core library will directly call process.stdout.write for commands
     // We don't want :: commands to be executed by the runner during tests
-    if (!str.match(/^::/)) {
+    if (!String(str).match(/^::/)) {
```
Comment (vsvipul) [inline jest.config.js:20]: jest new version was throwing error as type for str here was being recognized as "any". Implicitly converting to String fixed this.

File: .eslintrc.json [hunk 2]
```diff
@@ -9,15 +9,14 @@
...
-    "prettier/@typescript-eslint"
+    "plugin:prettier/recommended"
   ],
   "plugins": ["@typescript-eslint", "simple-import-sort", "jest"],
   "rules": {
     "import/first": "error",
     "import/newline-after-import": "error",
     "import/no-duplicates": "error",
-    "simple-import-sort/sort": "error",
+    "simple-import-sort/imports": "error",
```
Comment (vsvipul) [inline .eslintrc.json:19]: https://github.com/lydell/eslint-plugin-simple-import-sort/blob/main/CHANGELOG.md#version-600-2020-11-15

File: .eslintrc.json [hunk 3]
```diff
@@ -9,15 +9,14 @@
     "plugin:import/errors",
     "plugin:import/warnings",
     "plugin:import/typescript",
-    "plugin:prettier/recommended",
-    "prettier/@typescript-eslint"
+    "plugin:prettier/recommended"
```
Comment (vsvipul) [inline .eslintrc.json:12]: https://github.com/prettier/eslint-config-prettier/blob/main/CHANGELOG.md#version-800-2021-02-21

File: .gitattributes [hunk 4]
```diff
@@ -1 +1,2 @@
-.licenses/** -diff linguist-generated=true
\ No newline at end of file
+.licenses/** -diff linguist-generated=true
+* text=auto eol=lf
```
Comment (vsvipul) [inline .gitattributes:2]: windows-latest build was failing. To fix added this. See pt 4 at https://prettier.io/docs/en/options.html#end-of-line

File: jest.config.js [hunk 5]
```diff
@@ -17,7 +17,7 @@ const processStdoutWrite = process.stdout.write.bind(process.stdout);
...
         return processStdoutWrite(str, encoding, cb);
     }
 };
```
Comment (mkevre) [inline jest.config.js:23]:
```suggestion
        return processStdoutWrite(str, encoding, cb);
    }
 };
 ```

**Review by mkevre — APPROVED** [review]

**Comment by mkevre** [comment]
Duplicate of #

**Review by bishal-pdMSFT — APPROVED** [review]

**Comment by mkevre** [comment]
mkevre ***@***.***>, 31 Oca 2022 Pzt, 19:01 tarihinde şunu
yazdı:

**Pull request merged**